README.md

```diff
@@ -63,6 +63,7 @@ conda activate bsoid_v2
 You should now see (bsoid_v2) $yourusername@yourmachine ~ %
 
 #### Step 2: Run the pipeline through the CLI!
+##### Note: If running on an AWS instance, you will need to add a rule to your security group with Type: “Custom TCP Rule”, Port Range:8501, and Source: MyIP. Port 8501 is the custom port used by Streamlit.
```

Comment:
so you need to do this regardless, or to be able to view the streamlit launch in browser?

Reply:
Can you add that this is an inbound rule and that the python script runs the streamlit app? Also maybe add the context that the user will be able to use the external URL that b-soid prints to have a log of what's running -- if b-soid fails, the error messages appear there, not inline in the terminal, sometimes.

Reply:
What Taylor said. I'd note 1) if you are running this on AWS AND need to see the web application (which is useful for debugging since errors sometimes only appear in the web-app), you need to set inbound rule. 2) That the inbound rule port is 8501 but that the users should be careful to make sure that this is the right port when they run the app (for example, if you incorrectly shutdown so the process is running the background and you run the streamlit app again, it'll select the next port).

Reply:
This should not be an issue in VSCode technically but a useful note regardless.